### ISSUE TITLE
OSM map data version

### ISSUE THREAD

**Issue by Mesgary** [mapsme/omim#525]
I think it is a good idea that you show the user the date of the map. I mean that if you have downloaded map data directly from OSM server you can show this date, and if you have downloaded this from a data site like planet.osm, you can write the date of the planet.osm sync date with OSM.

**Comment by deathbaba** [comment]
It is a good idea, but it needs to fit into UI somehow. We’re completely refactoring Download UI right now, will see how it goes.

**Comment by MrZoidberg** [comment]
BTW it will be nice to have maps update not connected to Maps.Me app update. Also you can consider incremental updates to maps.

**Comment by deathbaba** [comment]
Thanks, we already have it in our TODO list :)

**Comment by MrZoidberg** [comment]
Great! Just FYI: please update Ukrainian map, there are a LOT of changes in street names.

**Comment by deathbaba** [comment]
Will do it soon, thanks!

**Comment by utack** [comment]
This is implemented in the "about" screen now.  
This issue can be closed?

**Comment by Mesgary** [comment]
No, the date which is displayed is build date of application. But I mean date of the map data. Now it is shown as:
Date 161214
Which should be changed to:
App Build Date 2016-12-14
Map Data Date .........-....-...

**Comment by deathbaba** [comment]
This date approximately corresponds to the day when data was taken from OpenStreetMap.org <http://openstreetmap.org/> and MAPS.ME data files were built.

**Comment by Mesgary** [comment]
No, I have not updated my maps and only updated the MAPS.ME application and not it shows the 161214 (which was changed after I updated the application). So it is the date of application.
As each map file may be updated separately, I think date of the map file should be shown on the map file list page.

**Comment by deathbaba** [comment]
Ok, to make it more correct: 161214 is the date of the OSM maps data which you get if “update” any old maps to the most current version.

**Comment by richlv** [comment]
each map can have its own date. this would either call for a separate row in "about" for each single map, or a way to see those in the map download section (probably more reasonable).
